WordPress/Classes/Extensions/UIImage+Exporters.swift

```diff
@@ -53,73 +53,73 @@ extension UIImage {
      - Parameters:
      - url: file url to where the asset should be exported, this must be writable location
      */
-    func writeJPEGToURL(url: NSURL) throws {
+    func writeJPEGToURL(_ url: URL) throws {
         let data = UIImageJPEGRepresentation(self, 0.9)
-        try data?.writeToURL(url, options: NSDataWritingOptions())
+        try data?.write(to: url, options: NSData.WritingOptions())
     }
 
     // Converts the imageOrientation from the image to the CGImagePropertyOrientation to use in the file metadata.
     var metadataOrientation: CGImagePropertyOrientation {
         get {
             switch imageOrientation {
-            case .Up: return CGImagePropertyOrientation.Up
-            case .Down: return CGImagePropertyOrientation.Down
-            case .Left: return CGImagePropertyOrientation.Left
-            case .Right: return CGImagePropertyOrientation.Right
-            case .UpMirrored: return CGImagePropertyOrientation.UpMirrored
-            case .DownMirrored: return CGImagePropertyOrientation.DownMirrored
-            case .LeftMirrored: return CGImagePropertyOrientation.LeftMirrored
-            case .RightMirrored: return CGImagePropertyOrientation.RightMirrored
+            case .up: return CGImagePropertyOrientation.up
```

Comment:
Since the return type already specifies `CGImagePropertyOrientation`, this can be reduced to:

```
case .up: return .up
case .down: return .down
case .left: return .left
case .right: return .right
case .upMirrored: return .upMirrored
case .downMirrored: return .downMirrored
case .leftMirrored: return .leftMirrored
case .rightMirrored: return .rightMirrored
```

PS: I know this wasn't introduced by this PR but I think it could be a good chance to do some simple cleanup.